src/pages/components/ToolBar.qml

```diff
@@ -168,15 +169,15 @@ Column {
                     onNewTabRequested: {
                         // New tab request triggers 90 degrees clockwise rotation
                         // for the tab icon.
-                        rotationAnimator.from = rotationAnimator.to
-                        rotationAnimator.to = rotationAnimator.from + 90
+                        rotationAnimator.from = 0
+                        rotationAnimator.to = 360
```

Comment:
Remove "easing.type: Easing.InOutQuad" from rotationAnimation to calm the animation. 360 InOutQuad rotation animation on 1.5sec doesn't look good.